rpp-charter.md

```diff
@@ -19,6 +19,10 @@ to be known collectively as RESTful Provisioning Protocol (RPP).
 These specifications will specifically target a REST architecture high on the Richardson Maturity Model
 using HTTPS and JSON. 
 
+The protocol shall incorporate extensibility considerations allowing to
```

Comment:
Is "shall incorporate" too commanding? Will the working group fail if it doesn't do this? Maybe the language should be softened to be "may consider".

Reply:
I think the to be formed WG shall cope with the extensibility issue. Not doing it would be narrowing scope too much and maybe event missing a point.
Would "shall consider extensibility capabilities" be more open as a command to consider rather than command to incorporate?

Reply:
IMHO, "shall consider extensibility capabilities" can be interpreted as a requirement to incorporate those features rather than to evaluate.

Maybe "will evaluate for possible inclusion". Though I think "may consider" is good enough.

Reply:
"will evaluate for possible inclusion" sounds good to me.